rpcs3/rpcs3qt/main_window.ui

```diff
@@ -638,9 +638,9 @@
     <string>RAP Files</string>
    </property>
   </action>
-  <action name="actionCheck_for_Updates">
+  <action name="updateAct">
    <property name="enabled">
-    <bool>false</bool>
+    <bool>true</bool>
```

Comment:
You can remove the enabled property as whole, the default value is true. Minor nitpick though.